open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -165,9 +159,63 @@ public void getOriginalTypes()
          * Calls for new and changed types go here
          */
         update0015LinkedMediaTypes();
+
+        add0042ProcessingStateClassification();
     }
 
 
+    /*
+     * -------------------------------------------------------------------------------------------------------
+     */
+
+
+    /**
+     * 0043  Add the Processing State Classification
+     */
+    private void add0042ProcessingStateClassification()
+    {
+        this.archiveBuilder.addClassificationDef(addProcessingStateClassification());
+    }
+
+    private ClassificationDef addProcessingStateClassification()
+    {
+
+        final String guid = "261fb0aa-b884-4ee8-87ea-a60510e9751d";
+        final String name = "ProcessingState";
+        final String description = "Stores last sync timestamp for critical elements.";
```

Comment:
The description needs to be more generic. The same can be used by different components and cases where critical element has different or no meaning.
I would suggest:
"Stores processing state information used by various SoftwareServerCapabilities"